content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md

```diff
@@ -92,5 +92,67 @@ Once there is information in the knowledge base, the agent will start using this
+
+By adding a column called **seachat_ref** in your spreadsheet, your agent will automatically extract information to add URL buttons to the agent's responses. You can put as many column as you want to add multiple URL buttons to the answer.
+
+| seachat_ref      ||
+|------------------|-|
+| button text |
+|button value |
+
+Since the agent will look into this particular column, we need to make sure that we use the correct variables to help the agent understand the information.
+
+**Please make sure to follow the format below to ensure the agent can extract the information correctly:**
+
+In each row of the **seachat_ref** column, your AI agent will look for two pieces of information: <code>button text</code> (line1) and <code>button value</code> (line2). The following is what your seachat_ref column should look like:
+
+```
+"User Manual Wiki" --- line 1
+"https://user-manual-wiki.com" --- line 2
+```
+
+<div id="additional-setting-ui" style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/knowledge-advanced-features/url-button/spreadsheet-example.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/knowledge-advanced-features/url-button/spreadsheet-example.png" alt="image showcasing how to write an agent description">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Spreadsheet Example</p>
+</div>
+
+> 🚨Attention🚨
+> 
+> If the button value is not a URL, e.g. some text, 2 paragraphs, or 3 URLs in 3 lines, the button will show the entire content of button value when clicked.
+
+All the information that you put inside `seachat_ref` will be extracted as buttons. If the content of the button value is a URL, the button will redirect the user to that URL when clicked. If the button value is not a URL, the button will return the button value in a response.
+
+<div id="additional-setting-ui" style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/knowledge-advanced-features/url-button/non-url-buttons.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/knowledge-advanced-features/url-button/non-url-buttons.png" alt="image showcasing how to write an agent description">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Buttons in Agent Response</p>
+</div>
+
+
+<br/>
+
+That's it. Now you no longer have to worry about manually adding URL buttons to your agent's responses when you are uploading a spreadsheet with a large amount of information.
```

Comment:
the spreadsheet screen's content doesn't match the chat window content. you can reduce spreadsheet and just use it to match the chat window content:

![image](https://github.com/seasalt-ai/wiki/assets/543547/b1f6a670-799c-4dce-80b1-334e6086a3ef)